EngineBay.Authentication/BasicAuth/CurrentIdentityFromBasicAuth.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace EngineBay.Authentication
+{
+    using EngineBay.Core;
+    using Microsoft.AspNetCore.Http;
+
+    public class CurrentIdentityFromBasicAuth : ICurrentIdentity
+    {
+        public CurrentIdentityFromBasicAuth(IHttpContextAccessor httpContextAccessor)
+        {
+            if (httpContextAccessor == null)
+            {
+                throw new ArgumentNullException(nameof(httpContextAccessor));
+            }
+
+            // TODO: Get user from database?
```

Comment:
I don't see this todo reflected on the Miro board?

Reply:
Added